### ISSUE TITLE
Dim on idle

### ISSUE THREAD

**Issue by Yeeheet** [xanderfrangos/twinkle-tray#261]
Hi !

I recently discovered your project is it's perfect, just what I was looking for. I just wish there was some kind of idle detection that would dim the screen when I'm away then go back to the last specified brightness. Don't know if that's in your plans so here's a suggestion !

Cheers and nice work 😄

**Comment by xanderfrangos** [comment]
@DRSDavidSoft That's correct. 👍 I haven't prioritized cleaning up old issues (as you've probably noticed).

**Comment by DRSDavidSoft** [comment]
@xanderfrangos That's quite alright, primary work comes first! 👍🏻  Thanks for keeping on developing awesome apps 😎
